### ISSUE TITLE
some types in vCard 3.0 not supported

### ISSUE THREAD

**Issue by steview2000** [jspricke/python-abook#3]
Hi,

After having some troubles, I figured that your script does not support certain type that are defined in vCard 3.0. That is in general not a problem, but it would be good if the relevant line in the vcf file is somehow imported into abook instead of ignored.

For example, I have a file with the following entry:
BEGIN:VCARD
VERSION:3.0
PRODID:
N:Max;Mustermann;;;
FN: Max  Mustermann        
TEL;type=pref:+12345566
ADR;type=WORK;type=pref:;;Musterstr.;;;;Germany
END:VCARD

Using vcf2abook ignores the phone number. 

Here is the specification for vCard 3.0:
https://www.evenx.com/vcard-3-0-format-specification

**Comment by jspricke** [comment]
As stated in the readme, the supported types are:
`view PHONE = phone, workphone, mobile, other`
https://github.com/jspricke/python-abook/blob/master/README.rst#L14

Which is already an extension to what Abook supports by default. We could add more types, but I'm not sure if adding all from the vcard specification makes sense. So if you come up with a sensible selection in a pull request, we could discuss it. Note that the intention of python-abook was to be close to what abook supports, not what vcard supports (which is a lot more and I'm not sure if we could tweak abook to support all).

**Comment by steview2000** [comment]
I don't think that more types are necessary.  I would add in abook.py after line 260:
```
            else:
                entry['phone'] = tel.value
```

It is so that the phone number is not ignored, just because the type is unknown.

**Comment by jspricke** [comment]
Than we would overwrite they `type=home` and every other type not handled before. I don't think that makes sense.
